### ISSUE TITLE
Ansible error in current master

### ISSUE THREAD

**Issue by str4d** [zcash/zcash-gitian#27]
This is from a clean start (after running `vagrant destroy zcash-build`). Hits this error, and subsequent runs repeat it.

```
$ vagrant up --provision zcash-build
Bringing machine 'zcash-build' up with 'virtualbox' provider...
==> zcash-build: Checking if box 'debian/jessie64' is up to date...
==> zcash-build: Running provisioner: ansible...
    zcash-build: Running ansible-playbook...
PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ForwardAgent=yes -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --limit="zcash-build" --inventory-file=/home/str4d/dev/zcash/zcash-gitian/.vagrant/provisioners/ansible/inventory -v gitian.yml
Using /home/str4d/dev/zcash/zcash-gitian/ansible.cfg as config file
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/home/str4d/dev/zcash/zcash-gitian/roles/common/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- include_tasks: repartition.yml
  ^ here


The error appears to have been in '/home/str4d/dev/zcash/zcash-gitian/roles/common/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- include_tasks: repartition.yml
  ^ here

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

```
$ ansible --version
ansible 2.3.1.0
  config file = /home/str4d/dev/zcash/zcash-gitian/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609]
```

vagrant-disksize (0.1.2)

**Comment by str4d** [comment]
Aha. I'd assumed it wasn't that because the README says `Tested with Ansible 2.1.2 and Vagrant 1.8.6 on Debian GNU/Linux (jessie).`

I was able to successfully continue by running the `sudo pip install -U ansible` command earlier on in the README (which installed 2.5), so as long as the minimum-required version is documented then it should be fine.

**Comment by str4d** [comment]
Closed by #31.
